### PR TITLE
Merge general SCONEPRO and video streaming deliverables

### PR DESCRIPTION
**Pull request by SpencerDawkins**
@fluffy and @martinthomson -This is based on our conversation today. I merged the two proposed standards-track documents, and moved discovery to the informational applicability and management document, as part of solution-specific considerations. 

Please advise if you were expecting something else!

p.s. @mjoras, I'm sure there's a way to add @fluffy and @martinthomson as reviewers for any PRs in the repo, but I don't know how to do that. When I tried to add them as reviewers for this PR, nothing came up for me to choose, so I ended up tagging them, but that's pretty error-prone, as a way of making sure they will see everything they need to review.

### CODE REVIEW

File: documents/charter.md [hunk 1]
```diff
@@ -57,20 +57,17 @@ The proposed deliverables for SCONEPRO are as follows:
...
 	* describe minimal reference architecture for SCONEPRO deployment
-    * discovery mechanism (if required)
-    * connection establishment, communication signaling and data format (container to share network information)
-    * specify and justify privacy and security requirements
-
-* Develop a standard track specification based on SCONEPRO protocol to communicate network properties applicable to streaming video from the SCONEPRO monitor to the client(s). Its contents might include
-
-    * describe use case and motivation
+	* connection establishment, communication signaling and data format (container to share network information)
+	* describe use case and motivation for SCONEPRO aupport of streaming video applications
```
Comment (martinthomson) [inline documents/charter.md:63]:
```suggestion
	* describe use case and motivation for SCONEPRO support of streaming video applications
```

**Review by martinthomson — COMMENTED** [review]
As I noted, the detail here is probably too much for a charter.  If there are important aspects to this, those sub-bullets should be integrated into the main charter text.

**Comment by SpencerDawkins** [comment]
> As I noted, the detail here is probably too much for a charter. If there are important aspects to this, those sub-bullets should be integrated into the main charter text.

@martinthomson, I agree with both of these observations. I'd propose a counter-offer, that we remove the sub-bullets from the charter and put them on a slide, either for the BOF or for the first working group meeting. 

I included this level of detail during spirited discussion in #11 and #14, [at Zahed's request](https://github.com/mjoras/SCONE-PROTOCL/pull/14#discussion_r1466496857). I don't think @zaheduzzaman intended the details to go into the version of the charter that the IESG ballots on - it was just in a convenient place that everyone would see, when we were working out the high level deliverables.

**Comment by martinthomson** [comment]
Having these on slides seems like a good idea, but recognize that this would be sliding into a working group discussion, not a working group *forming* discussion.  If people don't agree on the finer details, then we can suggest that this is a topic the working group needs to resolve.  The only reason to show this would be to demonstrate that there is a credible plan forming for how to tackle the problems, not to agree or resolve on them.

**Comment by SpencerDawkins** [comment]
Watch me agree violently with @martinthomson. 

> Having these on slides seems like a good idea, but recognize that this would be sliding into a working group discussion, not a working group _forming_ discussion. If people don't agree on the finer details, then we can suggest that this is a topic the working group needs to resolve. The only reason to show this would be to demonstrate that there is a credible plan forming for how to tackle the problems, not to agree or resolve on them.

I agree (violently). So, to move this PR forward, I will remove the sub-bullet details from the PR.

**Review by fluffy — APPROVED** [review]
LGTM

**Pull request merged**